src/igvfd/audit/measurement_set.py

```diff
@@ -492,6 +492,35 @@ def audit_missing_auxiliary_set(value, system):
                 yield AuditFailure(audit_message.get('audit_category', ''), f'{detail} {audit_message.get("audit_description", "")}', level=audit_message.get('audit_level', ''))
 
 
+@audit_checker('MeasurementSet', frame='embedded')
+def audit_missing_strand_specificity(value, system):
+    '''
+    [
+        {
+            "audit_description": "Single-cell CRISPR screens, such as Perturb-seq, scCRISPR screen, TAP-seq, and CERES-seq, are expected to specify strand specificity.",
+            "audit_category": "missing strand specificity",
+            "audit_level": "WARNING"
+        }
+    ]
+    '''
+    audit_message_strand_specificity = get_audit_message(audit_missing_strand_specificity)
+
+    expected_assays = {'Perturb-seq', 'scCRISPR screen', 'TAP-seq', 'CERES-seq'}
+    preferred_assay_title = value.get('preferred_assay_title')
+    strand_specificity = value.get('strand_specificity')
```

Comment:
Although this is okay I would recommend to define the default value like `value.get('strand_specificity', None)` so that it is explicit it will return "none".